src/PeerConnection.cpp

```diff
@@ -209,15 +209,16 @@ namespace mediasoupclient
 		MSC_TRACE();
 
 		webrtc::SdpParseError error;
-		webrtc::SessionDescriptionInterface* sessionDescription;
-		rtc::scoped_refptr<SetSessionDescriptionObserver> observer(
-		  new rtc::RefCountedObject<SetSessionDescriptionObserver>());
+		std::unique_ptr<webrtc::SessionDescriptionInterface> sessionDescription;
```

Comment:
this is not used anymore, right?

Reply:
yup its dead. apologies on the slow reply

Reply:
fixed in next commit @jmillan